### PR TITLE
Home work 2: Data types overview

### PR DESCRIPTION
**Pull request by mylak**

### CODE REVIEW

File: Exercises/data-types-overview/data-types-overview.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+// # Data types
+// 1. Declare a variable on each type using two options: plain creation, creation with constructor.
+
+// plain creation:
+let stringVariable = "Hello World!";
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: always use `const` when variable not going to be change

File: Exercises/data-types-overview/data-types-overview.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+let booleanVariable = true;
+let nullVariable = null;
+let undefinedVariable = undefined;
+let objectVariable = {
+  name: "John",
+  age: 30
+}
+
+// creation with constructor:
+let stringConstructorVariable = new String ("The string from constructor");
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: no space before `(`

File: Exercises/data-types-overview/data-types-overview.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+let numberConstructorVariable = new Number (777);
+let boolinConstructorVariable = new Boolean (false);
+let objectConstructorVariable = new Object ({company:"Ciklum"});
+
+// 2. Use operator typeof on each of them.
+
+// plain creation:
+console.log (typeof (stringVariable));
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: no need to wrap in  `(`

File: Exercises/data-types-overview/data-types-overview.js (hunk 2 above)
Comment (mylak) [reply Exercises/data-types-overview/data-types-overview.js]: fixed

File: Exercises/data-types-overview/data-types-overview.js (hunk 3 above)
Comment (mylak) [reply Exercises/data-types-overview/data-types-overview.js]: fixed

File: Exercises/data-types-overview/data-types-overview.js (hunk 1 above)
Comment (mylak) [reply Exercises/data-types-overview/data-types-overview.js]: fixed

File: Exercises/data-types-overview/data-types-overview.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+// creation with constructor:
+console.log (typeof stringConstructorVariable);
+console.log (typeof numberConstructorVariable);
+console.log (typeof boolinConstructorVariable);
+console.log (typeof objectConstructorVariable);
+
+// 3. Demonstrate object's references.
+
+let objOne = {
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: `const` preferable

File: Exercises/data-types-overview/data-types-overview.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+- we can use multi-line strings;
+- we can embed variables into the string: ${stringVariable}, ${booleanVariable};
+- we can even make some basic math operations: 5+3=${5+3};
+!`
+
+console.log (stringBackticks);
+
+// # Additional
+// 1. Declare an array containing different data types.
+let arrayVariables = ["string", 228, true, null, undefined, {name:"Jack", age: 23}];
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: `const` preferable

**Pull request merged**